mdata/cache/cache.go

```diff
@@ -0,0 +1,107 @@
+package cache
+
+import (
+	"flag"
+	"sync"
+
+	"github.com/raintank/metrictank/iter"
+	accnt "github.com/raintank/metrictank/mdata/cache/accnt"
+	"github.com/rakyll/globalconf"
+)
+
+var (
+	maxSize uint64
+)
+
+func init() {
+	flags := flag.NewFlagSet("chunk-cache", flag.ExitOnError)
+	// (1024 ^ 3) * 4 = 4294967296 = 4G
+	flags.Uint64Var(&maxSize, "max-size", 4294967296, "Maximum size of chunk cache in bytes")
+	globalconf.Register("chunk-cache", flags)
+}
```

Comment:
it does appear to be working. but if you prefer to keep that consistent with the other stuff I can move it into the `ConfigSetup()` callback too